src/sqlancer/doris/gen/DorisAlterTableGenerator.java

```diff
@@ -0,0 +1,70 @@
+package sqlancer.doris.gen;
+
+import sqlancer.Randomly;
+import sqlancer.common.query.ExpectedErrors;
+import sqlancer.common.query.SQLQueryAdapter;
+import sqlancer.doris.DorisErrors;
+import sqlancer.doris.DorisProvider.DorisGlobalState;
+import sqlancer.doris.DorisSchema.DorisCompositeDataType;
+import sqlancer.doris.DorisSchema.DorisTable;
+import sqlancer.doris.DorisToStringVisitor;
+
+public final class DorisAlterTableGenerator {
+
+    private DorisAlterTableGenerator() {
+    }
+
+    enum Action {
+        ADD_COLUMN, ALTER_COLUMN, DROP_COLUMN
+    }
+
+    public static SQLQueryAdapter getQuery(DorisGlobalState globalState) {
+        ExpectedErrors errors = new ExpectedErrors();
+        errors.add(" does not have a column with name \"rowid\"");
```

Comment:
Here and below, I noticed that the errors are the same as the current DuckDB implementation (see https://github.com/sqlancer/sqlancer/blob/6088b218d043a751e35c4056cb68dd22d3d5609c/src/sqlancer/duckdb/gen/DuckDBAlterTableGenerator.java#L23-L24). I assume that this statement is not yet supported (also judging from this provider)? If so, let's remove it from the PR - we can re-added in a subsequent one.